vernac/classes.mli

```diff
@@ -72,6 +72,18 @@ val declare_new_instance
 
 val add_class : typeclass -> unit
 
+type event =
```

Comment:
It wouldn't hurt to scope this (and the record fields) in a `Event` module, so this type becomes `Event.t`.

Reply:
@gares do you intend to do this?

Reply:
If you want me to, ok.
Not sure it is worth it if it is the only change to be made.

Reply:
Do as you prefer, we are not especially consistent about this anyway